### DISCUSSION TITLE
Range DatePicker doesn't change delimiter symbol with renderValue

### DISCUSSION THREAD

**Discussion by verbeckii** [rsuite/rsuite#2050] [Q&A] [answered]
Hello everyone

I'm trying to change the delimiter symbol between two dates with renderValue props
and it works fine on _codesandbox_ **but** it doesn't work on my project
here my test [codesandbox ](https://codesandbox.io/s/small-water-43798?file=/src/App.js)

Please see my attachment 
The delimiter symbol changed in calendar label **but** doesn't change on input (all works well in empty _codesandbox_ )
![image](https://user-images.githubusercontent.com/60220767/138868116-4850b275-9344-4ed5-8f95-a6121dbb7791.png)

So all I found is the difference between _Input_ tag and _span_
Please see the attachment
renderValue touched span tag... ( _codesandbox_ has the same behavior but everything works well)
![image](https://user-images.githubusercontent.com/60220767/138868538-cd63c763-499e-4832-aea1-4ef8afdf8fc7.png)

```
renderValue={([startDate, endDate]) => {
          return (
              format(startDate, 'MMM dd, yyyy') + ' - ' + format(endDate, 'MMM dd, yyyy')
          );
      }}
```

Sure I'm using the latest version of rsuite and react on my project and sandbox (rsuite and react versions are identical)

maybe you have some advice for me on how to solve this problem?

**Comment by simonguo** [comment, marked as answer]
Hi, please check the version of rsuite you are using first.
If you are using rsuite v5, you can directly use format and character to meet your needs.

```tsx
<DateRangePicker format="MMM dd, yyyy" character=" - "/>
```

![image](https://user-images.githubusercontent.com/1203827/139019108-420b0799-bf37-4851-b0c4-0d50888777aa.png)

**Reply by verbeckii** [reply]
thank you
It works, I can't find character in the props doc
